pkg/framework/context/context.go

```diff
@@ -43,3 +50,11 @@ func WithOutboundTransport(ot transport.OutboundTransport) ProviderOption {
 		return nil
 	}
 }
+
+// WithDIDResolverOpt injects transport provider into the framework
+func WithDIDResolverOpt(opt didresolver.Opt) ProviderOption {
```

Comment:
Similarly - https://github.com/hyperledger/aries-framework-go/pull/135/files#r315815228

Reply:
@troyronda Updated to configure DIDResolver rather than lower level configs.